src/main/webapp/js/instructorFeedbackEdit.js

```diff
@@ -294,19 +292,16 @@ function deleteQuestion(number) {
  * Allows users to cancel editing questions
  */
 function cancelEdit(number) {
-    if (number === -1) {
-        if (confirm('Are you sure you want to cancel adding this question?')) {
-            location.reload();
-        }
-    } else {
-        if (confirm('Are you sure you want to cancel your changes?')) {
-            location.reload();
-        }
+    var confirmationMsg = number === -1 ?
+            'Are you sure you want to cancel adding this question?' :
+            'Are you sure you want to cancel your changes?';
```

Comment:
I assume you are referring to the position of operators in multiple lines statements?
A glance through the top results for a search of "JS style guides" shows both in use.
While Google's is probably the most followed (being Google and all), there are several guides that recommend operators in front as well.

For our case, it may be good following the same format as Java for standardisation.
(And there is much less JavaScript code to update compared to Java.)